index.js

```diff
@@ -1,17 +1,27 @@
 const axios = require('axios')
+const isServer = require('detect-node')

```

Comment:
I don't really like the design of this. A alternative design would be:
1. make the same file with a `createClientCreator`  function (funny name but that what it is). This function would get a `correlate` and logging functions (which might do nothing). 
2. make two main files one for browser and one for server. In the server we will pass the correlate function and the logging functions and in the browser file we will pass empty function (for now, in the future we might change those function).
3. In the package json you can add the `browser` field to change the index file when running in the browser.

Reply:
Why not use the `is-server` package?

Reply:
I'm not talking about builder, I'm talking about a factory of the factory. The idea is to create polymorphism for the `correlate` and the logging functions.

Reply:
They have, they get options as an argument.
If you create an object, of which that some purpose is to create another object, it's a builder.